### DISCUSSION TITLE
[LspSaga] Any way to get the fullpath of the file that contains a definition in the peek mode?

### DISCUSSION THREAD

**Discussion by AlgusDark** [nvimdev/lspsaga.nvim#1100] [Q&A] [answered]
Right now, if the title is available, we get the name of the file, but I would like to display a full path. It helps me more to see "path/to/the/file.lua" instead of "file.lua".

**Comment by glepnir** [comment, marked as answer]
it already support in new version. need time to complete all features . please wait for it .thanks

**Reply by AlgusDark** [reply]
Thank you for this awesome project :D
